### PR TITLE
[SPEC] Fix link, improve clarity in docs

### PR DESCRIPTION
**Pull request by nonparibus**

### CODE REVIEW

**Review by kachkaev — APPROVED** [review]
Screenshot 2022-02-01 at 19 30 22

[Looks better!](https://blockprotocol-4msyujsx4-hashintel.vercel.app/docs/faq#how-does-the-block-protocol-relate-to-graphql) FYI: We still have `[` `]` around _application_ (not sure I get the meaning of them).

**Comment by nonparibus** [comment]
> Screenshot 2022-02-01 at 19 30 22
> 
> [Looks better!](https://blockprotocol-4msyujsx4-hashintel.vercel.app/docs/faq#how-does-the-block-protocol-relate-to-graphql) FYI: We still have `[` `]` around `application` (not sure I get the meaning of them).

I also wasn't sure. @CiaranMn?

**Comment by CiaranMn** [comment]
> > Screenshot 2022-02-01 at 19 30 22
> > [Looks better!](https://blockprotocol-4msyujsx4-hashintel.vercel.app/docs/faq#how-does-the-block-protocol-relate-to-graphql) FYI: We still have `[` `]` around `application` (not sure I get the meaning of them).
> 
> I also wasn't sure. @CiaranMn?

The sentence is a quote from the linked page, except "application" which is replacing a word

File: site/src/_pages/docs/4_faq.mdx [hunk 1]
```diff
@@ -127,7 +127,7 @@ In theory, the Block Protocol could use JSON-LD as the format in which entities
 
 ### How does the Block Protocol relate to GraphQL?
 
-GraphQL provides a _[syntax and system for describing_ [application] _data requirements and interactions_](https://spec.graphql.org/October2021/#sec-Overview), whereas the Block Protocol is specifying a particular set of interactions: those between a block and any application embedding it.
+GraphQL provides a [syntax and system for describing [application] data requirements and interactions](https://spec.graphql.org/October2021/#sec-Overview), whereas the Block Protocol is specifying a particular set of interactions: those between a block and any application embedding it.
```
Comment (kachkaev) [inline site/src/_pages/docs/4_faq.mdx:130]:
```suggestion
GraphQL provides a [“syntax and system for describing [application] data requirements and interactions”](https://spec.graphql.org/October2021/#sec-Overview), whereas the Block Protocol is specifying a particular set of interactions: those between a block and any application embedding it.
```

(if it’s a quote)

**Review by kachkaev — DISMISSED** [review]

**Review by CiaranMn — APPROVED** [review]

**Pull request merged**